### PR TITLE
HPCC-31872 Update Docs for Cost Tracking improvements

### PR DESCRIPTION
**Pull request by g-pan**
## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

## Testing:

All Unit tests Passed: https://github.com/g-pan/github-action-dev-build/actions/runs/10474453301

### CODE REVIEW

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml [hunk 1]
```diff
@@ -961,7 +972,15 @@ thor: []
           <graphic fileref="../../images/SCOST_img3_1.jpg"><!--ADD-IMAGE--></graphic>
         </sect4>
       </sect3>
+<sect3 id="Compile_Costs">
+        <title>Compile Cost</title>
+
+        <para>Compile costs are the costs of compiling the ECL code. The cost
+        of compiling the ECL code is included as a column in the workunit list. In the workunit summary page 
+        there is compile cost field.</para>
```
Comment (JamesDeFabia) [inline docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml:980]: In the workunit summary page, there is a compile cost field.

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml [hunk 2]
```diff
@@ -1003,16 +1028,18 @@ thor: []
...
-        in the summary page.</para>
-
-        <para>The new cost field is shown in the Logical File summary page. It
-        is the combined cost of storing and accessing the data.</para>
-
-        <para>The cost information is currently only generated for Thor and
-        hThor jobs.</para>
-      </sect3>
+        the graph, the subgraph and the workflow scope level.
+        Notable improvement to the File Access Cost is files that are accessed from the page cache doesn’t incur file access cost.
```
Comment (JamesDeFabia) [inline docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml:1032]: this sentence is not needed:
Notable improvement to the File Access Cost is files that are accessed from the page cache doesn’t incur file access cost.

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml [hunk 3]
```diff
@@ -1003,16 +1028,18 @@ thor: []
...
-
-        <para>The new cost field is shown in the Logical File summary page. It
-        is the combined cost of storing and accessing the data.</para>
-
-        <para>The cost information is currently only generated for Thor and
-        hThor jobs.</para>
-      </sect3>
+        the graph, the subgraph and the workflow scope level.
+        Notable improvement to the File Access Cost is files that are accessed from the page cache doesn’t incur file access cost.  
+        The HPCC System now detects data that has been returned from the page cache and adjusts cost calculations appropriately to produce more accurate cost calculations.
```
Comment (JamesDeFabia) [inline docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml:1033]: Cost Tracking detects data...

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml [hunk 4]
```diff
@@ -1003,16 +1028,18 @@ thor: []
...
-        <para>The new cost field is shown in the Logical File summary page. It
-        is the combined cost of storing and accessing the data.</para>
-
-        <para>The cost information is currently only generated for Thor and
-        hThor jobs.</para>
-      </sect3>
+        the graph, the subgraph and the workflow scope level.
+        Notable improvement to the File Access Cost is files that are accessed from the page cache doesn’t incur file access cost.  
+        The HPCC System now detects data that has been returned from the page cache and adjusts cost calculations appropriately to produce more accurate cost calculations. 
+        These file access costs for a workunit is recorded with the workunit and displayed
```
Comment (JamesDeFabia) [inline docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml:1034]: SV agreement: costs **are** recorded

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml [hunk 5]
```diff
@@ -1003,16 +1028,18 @@ thor: []
...
+        in the summary page and the workunit’s metrics.</para>
+
+      <sect4 id="FileCostAtRest"> <title>File Cost at Rest</title>
+
+        <para>The file cost at rest field is shown in the Logical File summary page. It
```
Comment (JamesDeFabia) [inline docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml:1039]: The **File Cost At Rest** field

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
Few comments inline

**Comment by g-pan** [comment]
Incorrectly linked to the wrong JIRA. Killing this PR to re-issue correctly.

**Pull request closed without merging**